### DISCUSSION TITLE
Usb tethering no internet connection but wifi tethering works

### DISCUSSION THREAD

**Discussion by ilsorliamsi** [Mygod/VPNHotspot#258] [Q&A]
Device Info
------------
Redmi Note 4 Snapdragon 625
Android Nougat 7.1.2 Lineageos 14.1

My other device working fine Samsung Galaxy Note 3 Snapdragon 800 with same lineageos version.

[vpnhotspot-869004223.log](https://github.com/Mygod/VPNHotspot/files/6932809/vpnhotspot-869004223.log)

**Comment by Mygod** [comment]
Seems working as expected. Try again?

**Comment by Mygod** [comment]
Also try reapply routing rules.

**Comment by ilsorliamsi** [comment]
> Also try reapply routing rules.

it still not working. it happen in MIUI stock rom too. maybe this app is not compatible with snapdragon 625 soc.

**Comment by Mygod** [comment]
I've moved this issue to discussions since it seems device specific.
